src/schema/fair.ts

```diff
@@ -56,6 +70,29 @@ const FairType = new GraphQLObjectType({
     about: {
       type: GraphQLString,
     },
+    followed_content: {
+      type: FollowedContentType,
+      resolve: (
+        fair,
+        _options,
+        _request,
+        { rootValue: { followedArtistsLoader, followedPartnersLoader } }
+      ) => {
+        const fair_id = fair.id
+
+        return {
+          artists: followedArtistsLoader({ fair_id }).then(({ body }) => {
+            return body.map(artist_follow => artist_follow.artist)
+          }),
+          galleries: followedPartnersLoader({
+            fair_id,
+            owner_types: ["PartnerGallery"],
+          }).then(({ body }) => {
+            return body.map(profile_follow => profile_follow.profile.owner)
+          }),
+        }
```

Comment:
is it worth thinking about the cases when you may only want artists or galleries, or do we always used both?

Reply:
As far as I know this is a very specific use case that requires both, but I see your point - ought this be further decomposed?